rosdep/python.yaml

```diff
@@ -5867,6 +5851,19 @@ python3-ruamel.yaml:
 python3-schedule:
   debian: [python3-schedule]
   ubuntu: [python3-schedule]
+python3-scipy:
+  arch: [python-scipy]
+  debian: [python3-scipy]
+  fedora: [scipy]
```

Comment:
This key isn't correct on Fedora; that will actually get you the python2 version of it.  This key should be `python3-scipy`.  While we are at it, we may as well change the `python-scipy` key for Fedora to be `python2-scipy`, as it is more descriptive.

Reply:
Thanks for the clarification, @clalancette. I believe I saw this when looking for the packages but I was worried about changing the `python-scipy` entry and disrupting older versions of Fedora where python == python2. I've implemented these changes, though.